README.md

```diff
@@ -3,10 +3,19 @@ Python script for creating new projects in the desired local directory, with a G
   
 Contact us [on Discord.](https://discord.gg/eqWstJu)
 ## Requirements:
+#### Universal
 - Python 3.4+
 - Git
-- npm
-- A text editor (i.e. VS Code, Atom, etc.)
+- (Optional but recommended) A text editor (i.e. VS Code, Atom, etc.)
+In addition:
+#### Node/React
+- NPM
+- (React) create-react-app
```

Comment:
comes with npm, not required

Reply:
this needs to go too..... it is part of npx which comes with npm https://reactjs.org/docs/create-a-new-react-app.html

Reply:
Do you mean that the script comes with NPM, or that CRA comes with NPM

Reply:
Where did NPM go?